tfhe/src/high_level_api/integers/keys.rs

```diff
@@ -35,24 +35,28 @@ impl IntegerConfig {
 
     pub(in crate::high_level_api) fn default_big() -> Self {
         Self {
-            block_parameters: Some(crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS),
+            block_parameters: Some(
+                crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS.into(),
+            ),
             wopbs_block_parameters: None,
         }
     }
 
     pub(in crate::high_level_api) fn default_small() -> Self {
         Self {
-            block_parameters: Some(crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_PBS_KS),
+            block_parameters: Some(
+                crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_PBS_KS.into(),
+            ),
             wopbs_block_parameters: None,
         }
     }
 
     pub fn enable_wopbs(&mut self) {
         let block_parameter = self
             .block_parameters
-            .get_or_insert(crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS);
+            .get_or_insert(crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS.into());
 
-        let wopbs_block_parameters = match block_parameter.encryption_key_choice {
+        let wopbs_block_parameters = match block_parameter.encryption_key_choice() {
             EncryptionKeyChoice::Big => crate::shortint::parameters::parameters_wopbs_message_carry::WOPBS_PARAM_MESSAGE_2_CARRY_2_KS_PBS,
             EncryptionKeyChoice::Small => crate::shortint::parameters::parameters_wopbs_message_carry::WOPBS_PARAM_MESSAGE_2_CARRY_2_KS_PBS,
```

Comment:
just looks a bit odd, but I don't think we have wopbs params for small params, so maybe error out on the small choice for now ?